mathjax3-ts/input/tex/ams_cd/AmsCdMethods.ts

```diff
@@ -47,14 +47,15 @@ let AmsCdMethods: Record<string, ParseMethod> = {};
 AmsCdMethods.CD = function(parser: TexParser, begin: StackItem) {
   parser.Push(begin);
   let item = parser.itemFactory.create('array') as ArrayItem;
+  let options = parser.configuration.options.amsCd;
```

Comment:
Somehow I feel there should be a functional way of retrieving these options. Something like "global" vs "local" options of the package. But I think it's good enough as it is, for now.

E.g., each package should get a unique name, via the configurations, which should then automatically disambiguate names for options, etc.  This could then also be used in to avoid naming conflicts of `handlers`.

Reply:
Sounds like a good plan.

I'd also like to see a configuration-specific data storage area (for things like the color model in the `color` extension) that can be used for data that is specific to the instance of the input jax holding the configuration.  Right now this is being added to the options, as really the only possibility at the moment, but that is a hack.